web/src/compose_recipient.js

```diff
@@ -95,6 +96,27 @@ function update_fade() {
 export function update_on_recipient_change() {
     update_fade();
     update_narrow_to_recipient_visibility();
+    update_compose_draft_count();
+}
+
+export function update_compose_draft_count() {
+    if (!compose_state.has_full_recipient()) {
+        return;
```

Comment:
Doesn't this need to clear it to `""`?

Reply:
My bad, I'll add a line clearing it before returning